openapi/components/schemas/RuleActionScheduleReminder.yaml

```
@@ -0,0 +1,30 @@
+allOf:
+  - $ref: ./RuleAction.yaml
+  - type: object
+    properties:
+      role:
+        type: string
+        enum:
+          - all
+          - renewal
+          - trial-end
+        description: Role of Reminder (available only on Order events, other events should use `all`).
```

suggestion: I found the term 'role' confusing here
```suggestion
        description: |-
          Specifies the types of events that the reminder is applicable to.
          The `renewal` and `trial-end` options are only available for order events.
          Other events must use the `all` option.
```